modules/microshift-install-rpms-gitops.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-rpms-for-gitops_{context}"]
+== Installing the GitOps ArgoCD manifests from an RPM package
+
+You can use a lightweight version of the OpenShift GitOps with {microshift-short} to help manage your applications. Install the necessary ArgoCD manifests using a RPM package. 
+
+.Procedure
+
+. Install the GitOps packages by running the following command: 
++
+[source,terminal]
+----
+$ sudo dnf install microshift-gitops microshift-gitops-release-info
+----
+
+. To apply the manifest from the package to an active cluster, run the following command: 
```

Comment:
I think "applying manifest" is an implementation detail. What we mean to say here is that we just need to restart microshift service to complete GitOps installation.